### ISSUE TITLE
Mongols: Units becoming stuck around Silver Tree after destruction

### ISSUE THREAD

**Issue by rbrash1287** [Woprok/AOE4-AdvancedGameSettings#57]
This has happened quite a few times that I have kept up with, and figured it was time to try addressing it for possible solutions.
I have only saw this happen on games with AGS and Forest Nothing, in which if a Silver Tree landmark is destroyed, sometimes traders going to and from it will become stuck.  This issue will become so bad that it bogs down game for every player, reducing fps to less than 5, and gets worse until people crash or the entire game crashes.  

There is a way to fix this, either by killing stuck traders, or deleting them if they are yours, or repairing the ST fully. 

Not sure if this would be an issue with AGS or just a bug that relic would have to patch?

**Comment by Woprok** [comment]
It's definitely something Relic will have to patch, if they can figure out why it's happening. This is unlikely to be caused by AGS as I don't affect pathfinding in any way with my mod. 

It's possible that it might be caused by Forest Nothing due to fact that forest occupies most of the map and pathfinding is always wonky on that map...

**Comment by rbrash1287** [comment]
Yeah that's pretty much exactly what I'm leanin' towards too.  I've followed you since you started AGS here on Github, and have learned a good bit from just observing your progress working out the kinks and whatnot. 

I just wish Forest Nothing's creator would set up something similar to what you have here on ghub.  Honestly, I'm not sure if he's even reachable. With that map and AGS running together, they push aoe to the absolute limit, and so it's been a fun ride, but also exposed plenty of issues that I hope relic can get around to.
